Contrib/AffiliationServer/revenue.fsx

```diff
@@ -0,0 +1,118 @@
+#r "nuget:fstoolkit.errorhandling"
+#r "nuget:nbitcoin"
+#r "../../WalletWasabi/bin/Debug/net7.0/WalletWasabi.dll"
+#r "System.Security.Cryptography.dll"
+
+open System.Security.Cryptography
+
+open System.Text
+open NBitcoin
+open NBitcoin.RPC
+open System
+open System.IO
+open Newtonsoft.Json
+open FsToolkit.ErrorHandling
+open WalletWasabi.Affiliation.Models
+open WalletWasabi.Affiliation.Serialization
+open WalletWasabi.Affiliation.Models.CoinJoinNotification
+
+module CommandLine =
+    let tryParseArgument (arg : string) =
+        match arg.Split([| '=' |]) with
+        | [| k; v |] -> Some (k, v)
+        | _ -> None
+
+    let getArg name =
+        Environment.GetCommandLineArgs()
+        |> Array.map tryParseArgument
+        |> Array.choose id
+        |> Array.tryFind (fun (argName, _) -> argName = name )
+        |> Option.map snd
+
+    let getArgOrDefault name defaultValue =
+        getArg name
+        |> Option.defaultValue defaultValue
+
+    let getArgOrFail name =
+        match getArg name with
+        | Some value -> value
+        | None -> failwith $"Mandatory argument {name} was not provided"
+
+module Signature =
+
+    let verifySignature (pubkey:byte[]) (message:byte[]) signature =
+        let ver = System.Security.Cryptography.ECDsa.Create()
+        let mutable bytesRead = 0
+        ver.ImportSubjectPublicKeyInfo(ReadOnlySpan(pubkey) , &bytesRead)
```

Comment:
The method fails in case the pubkey is not good so, it makes no sense to check how many bytes it read.
Instead of `do` i prefer to do `let _` =  to make clear it returns something that we don't care about.

Reply:
done.